README.md

```diff
@@ -1,82 +1,479 @@
+
+- 4 vCPUs.
+- 8GiB of RAM.
+- SSD disk with 10GiB of free space.
+- Reliable and fast internet connection.
+- Static IP.
+
+Hardware requirements might change with time, but this setup is probably over provisioned in the current state. We’re planning to do a stress testing benchmark suite to understand and predict the behavior of the validator under different loads to have more data about potential future recommended system requirements.
+
+### Firewall configuration
+
+If you’re behind a firewall, you should open ports `:8080` or `:443`, depending on if you run with TLS certificates. By default, TLS is not required, thus, expecting `:8080` to be open to the external world.
+
+### System prerequisites
+
+There are two prerequisites for running a validator:
+
+- Install host-level dependencies.
+- Get EVM node API keys.
+
+Tableland has two separate networks:
+
+- `mainnet`: this network syncs mainnet EVM chains (e.g., Ethereum mainnet, Arbitrum mainnet, etc.).
+- `testnet`: this network is syncing testnet EVM chains (e.g., Ethereum Sepolia, Arbitrum Goerli, etc.).
+
+This guide will focus on running the validator in the `mainnet` network.
+
+We do this for two reasons:
+
+- The `mainnet` network is the most stable one and is also where we want the most number of validators.
+- We can provide concrete file paths related to `mainnet` and avoid being abstract.
+
+We’ll also explain how to run a validator using Alchemy as a provider for the EVM node API the validator will use. The configuration will be analogous if you use self-hosted nodes or other providers.
+
+#### Install host-level dependencies
+
+To run the provided docker-compose setup, you’ll need to have installed:
+
+- `git`: [Installation guide](https://github.com/git-guides/install-git).
+- `docker` with the Compose plugin: [The default Docker engine installation](https://docs.docker.com/engine/install/) already includes the Compose plugin (i.e., `docker compose` command).
+
+Note that there’s no need for a particular `Go` installation since binaries are compiled within a docker container containing the correct Go compiler versions. Despite not being strictly necessary, creating a separate user in the host is usually recommended to run the validator.
+
+#### Create EVM node API keys
+
+The current setup needs one API key per supported chain. The default setup expects Alchemy keys for the following: Ethereum, Optimism, Arbitrum One, and Polygon. Ankr is used for Filecoin, and QuickNode for Arbitrum Nova. But, you are free to use a self-hosted node or another provider that supports the targeted chains.
+
+To get your Alchemy keys, create an [Alchemy](https://alchemy.com) account, log in, and follow these steps:
+
+1. Create one app for each chain using the `+ Create App` button.
+2. You’ll see one row per chain—click the `View Key` button and copy/save the `API KEY`.
+
+To get your Ankr Filecoin Keys, create an [Ankr](http://ankr.com) account, log in, and do the following:
```

Comment:
We actually stopped using Ankr because they didn't support Calibration net and we are not live with Filecoin mainnet yet. For Calibration we are using Glif.io.

Reply:
@avichalp ahh, that's right. so, i can probably do 2 things:
1) make a callout that Tableland is not live on FIL mainnet yet; this key is shown for demo purposes because when we launch, Ankr is the provider we recommend using (right? iirc, they have the archival support we need)
2) make a callout that for FIL testnet, we use Glif.io. but, they allow for unauthenticated requests, so there's no need to set up a variable like `VALIDATOR_GLIF_FILECOIN_CALIBRATION_API_KEY`

Reply:
so, i think: it would be good to mention that we are not live on mainnet yet. but idk if we want to recommend Ankr yet. i would wait for them to offer the service (full archival node) we need. also, it may happen that Glif.io offers the archival node first (they plan to support it as soon as Lotus fix is ready and have offered us an API key to use it). 

also, for the current configuration, we can recommend leaving the Glif API key empty: eg: `VALIDATOR_GLIF_FILECOIN_CALIBRATION_API_KEY=`